src/DataGrid.tsx

```diff
@@ -221,21 +221,34 @@ function DataGrid<R, SR>(
     onFill,
     onPaste,
     // Toggles and modes
-    enableFilterRow = false,
-    cellNavigationMode = 'NONE',
-    enableVirtualization = true,
+    enableFilterRow,
+    cellNavigationMode: rawCellNavigationMode,
+    enableVirtualization,
     // Miscellaneous
-    editorPortalTarget = body,
+    editorPortalTarget: rawEditorPortalTarget,
     className,
     style,
     rowClass,
     // ARIA
     'aria-label': ariaLabel,
     'aria-labelledby': ariaLabelledBy,
     'aria-describedby': ariaDescribedBy
-  }: DataGridProps<R, SR>,
+  }: DataGridProps<R, SR, K>,
   ref: React.Ref<DataGridHandle>
 ) {
+  /**
+   * defaults
+   */
+  rowHeight ??= 35;
+  headerRowHeight ??= typeof rowHeight === 'number' ? rowHeight : 35;
+  headerFiltersHeight ??= 45;
+  const summaryRowHeight = rawSummaryRowHeight ?? (typeof rowHeight === 'number' ? rowHeight : 35);
```

Comment:
There is a little bit of nuance here vs default params, if a param isn't defaulted in the parameter list, then it keeps its original type in functions, as TS cannot know that those functions are only called after the `??=` line. I wish we could use `??=` in the param list.
```ts
function fnA(param = 1) {
  param; // type; number

  function test() {
    param; // type: number
  }
}

function fnB(param: number | undefined | null) {
  param; // type: number | undefined | null
  param ??= 1;
  param; // type: number

  function test() {
    param; // type: number | undefined | null
    // this is because we could call `test` before the line where we set it to 1
  }
}
```